src/mapgen/mg_biome.h

```diff
@@ -247,6 +247,6 @@ class BiomeManager : public ObjDefManager {
 private:
 	BiomeManager() {};
 
-	Server *m_server;
+	Server *m_server = nullptr;
```

Comment:
FYI: Superfluous due to initializer list in the source file. This is a comment and not necessary a change request.